server/migrations/20190219055021-create-share.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Shares', {
+      articleId: {
+        type: Sequelize.STRING,
```

Comment:
This is a great job @Princesso. Is there a reason why the articleId has type string and not UUID? Also I think it should be a foreign key referencing the primary key in the article table. Kindly check this out.

Reply:
Thanks Peace, it was an oversight. I have made changes to the file